docs/cog_guides/audio.rst

```diff
@@ -0,0 +1,3726 @@
+.. _audio:
+
+=====
+Audio
+=====
+
+This is the cog guide for the audio cog. You will
+find detailed docs about usage and commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+    	[p]load audio
+
+
+You can see additional help for any command in this guide by using ``[p]help`` with the
+command name, like ``[p]help playlist append``.
+
+In this guide, you will see references to "Lavalink" or the "Lavalink.jar". Lavalink is the
+Java-based audio backend we use to be able to play music through the bot. Most users will
+not have to worry much about Lavalink or what it is, as Audio manages this process for you
+by default. Advanced users can read more about Lavalink and special cases under the Lavalink - Advanced Usage section below.
+
+You will also see references to ``internal`` or ``external`` in regards to the Lavalink.jar.
```

Comment:
You shouldn't not be using internal and external.

the correct reference terms are managed and unmanaged node I'm 99% sure the pr making that change had been merged into develop